xbmc/guilib/guiinfo/MusicGUIInfo.cpp

```diff
@@ -99,16 +100,25 @@ bool CMusicGUIInfo::GetLabel(std::string& value, const CFileItem *item, int cont
         return true;
       case PLAYER_TITLE:
         value = tag->GetTitle();
+        if (!tag->GetStationName().empty() && tag->GetTitle().empty())
+        {
+          value = "";
```

Comment:
`value.clear()` is more efficient than `= ""`

Reply:
Thanks!  I've updated the code accordingly.